src/components/media-views.js

```diff
@@ -79,19 +79,27 @@ async function createGIFTexture(url) {
  * @param {string} src - Url to a video file.
  * @returns {Element} Video element.
  */
-function createVideoEl(src) {
+async function createVideoEl(src) {
   const videoEl = document.createElement("video");
   videoEl.setAttribute("playsinline", "");
   videoEl.setAttribute("webkit-playsinline", "");
   videoEl.loop = true;
   videoEl.crossOrigin = "anonymous";
-  videoEl.src = src;
+
+  if (!src.startsWith("webrtc://")) {
```

Comment:
hmm, this made up protocol is a bit concerning, if we are going to make it up we might as well it something that is clearly app specific like `hubs://[hubid]/[clientId]/video`

Reply:
getting the hub id in these places was going to add more incidental complexity than i was comfortable with, so for now i went with `hubs://clients/<client id>/video` -- these URLs are not saved anywhere so its probably OK for now. i agree having the full hub id would be more correct